spec/handlers/sessions_create_shared_examples.rb

```diff
@@ -19,20 +18,20 @@
       end
     end
 
-    context "oauth response doesn't match log in username/email" do
-      let(:authentication) { FactoryBot.create(:authentication, provider: 'google_oauth2') }
-      let(:login_providers) { { google_oauth2: { uid: 'some_other_uid'} } }
-
-      it "returns mismatched_authentication status and does not log in" do
-        result = nil
-        expect do
-          result = handle(
-            request: MockOmniauthRequest.new(authentication.provider, authentication.uid, {})
-          )
-        end.not_to change { user_state.signed_in? }
-        expect(result.outputs.status).to eq :mismatched_authentication
-      end
-    end
+    # context "oauth response doesn't match log in username/email" do
+    #   let(:authentication) { FactoryBot.create(:authentication, provider: 'google_oauth2') }
+    #   let(:login_providers) { { google_oauth2: { uid: 'some_other_uid'} } }
+    #
+    #   it "returns mismatched_authentication status and does not log in" do
+    #     result = nil
+    #     expect do
+    #       result = handle(
+    #         request: MockOmniauthRequest.new(authentication.provider, authentication.uid, {})
+    #       )
+    #     end.not_to change { user_state.signed_in? }
+    #     expect(result.outputs.status).to eq :mismatched_authentication
+    #   end
+    # end
```

Comment:
Maybe this is important to test?

Reply:
I think this is replaced with this spec: https://github.com/openstax/accounts/blob/main/spec/handlers/oauth_callback_spec.rb

But let me reenable and see what happens (think it's going to fail on that pesky `signing_up` method).

Reply:
it does, and this is in the `signing_up` context, so that's why.. going to keep here for reference though

Reply:
the `logging_in` specs are still running